src/lib/requestManager.ts

```diff
@@ -1,9 +1,24 @@
-import axios, { Method as HTTPMethod, ResponseType, AxiosRequestConfig, AxiosResponse } from 'axios';
+import axios, { Method as HTTPMethod, ResponseType, AxiosRequestConfig, AxiosResponse, AxiosError } from 'axios';
 
-export const defaultOptions: { responseType: ResponseType } = {
-  responseType: 'json',
+import { config } from 'config';
+import { JSONValue, keysToCamelCase } from 'helpers/json';
+
+export const successResponseInterceptor = (response: AxiosResponse<unknown>): AxiosResponse<unknown> => {
+  const responseData = response.data as JSONValue;
+  const formattedData = keysToCamelCase(responseData);
+  response.data = formattedData;
+
+  return response;
 };
 
+export const defaultOptions = (): { responseType: ResponseType; baseURL: string; headers?: { [key: string]: string } } => ({
+  responseType: 'json',
+  baseURL: `${config().apiBaseUrl}/api/v1`,
```

Comment:
let's put this file or function under `v1` sub-dir
- `src/lib/request/v1/requestManager.ts`
- or `src/lib/request/requestManager.ts` for core config and `src/lib/request/v1/config.ts` for v1 path routing